e2e/runner/runner.go

```diff
@@ -33,17 +33,24 @@ import (
 	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
 )
 
+type E2ERunnerOption func(*E2ERunner)
```

Comment:
NIT : `E2ERunnerWithOptions` or `E2ERunnerOptions`

Reply:
I don't think this should be plural unless it was a slice type or a struct with multiple members.

It's singular but becomes plural when you use it as a varadic argument.

Based on a quick survey, this is typically singular:
- https://medium.com/@omid.jn/the-option-pattern-in-go-9edf55e88320
- https://rednafi.com/go/dysfunctional_options_pattern/
- https://www.yellowduck.be/posts/the-functional-options-pattern-in-go